packages/bbui/src/Form/Core/Picker.svelte

```diff
@@ -71,105 +71,73 @@
   }
 </script>
 
-<button
-  {id}
-  class="spectrum-Picker spectrum-Picker--sizeM"
-  class:spectrum-Picker--quiet={quiet}
-  {disabled}
-  class:is-invalid={!!error}
-  class:is-open={open}
-  aria-haspopup="listbox"
-  on:mousedown={onClick}
->
-  {#if fieldIcon}
-    <span class="icon-Placeholder-Padding">
-      <img src={fieldIcon} alt="icon" width="20" height="15" />
-    </span>
-  {/if}
-
-  <span
-    class="spectrum-Picker-label"
-    class:is-placeholder={isPlaceholder}
-    class:auto-width={autoWidth}
+<div use:clickOutside={() => (open = false)}>
```

Comment:
I was worried that adding another wrapper div around the core Picker could have consequences for controlling the width, but I checked out the branch and everything still seems to work. If we see any instances of pickers not using the full width any more then we might need to add some styles here to ensure the inner button takes up the full width of this new wrapper div, but it seems fine for now.